packages/suricata/data_stream/eve/elasticsearch/ingest_pipeline/default.yml

```
@@ -5,6 +5,18 @@ processors:
   - set:
       field: ecs.version
       value: '8.8.0'
+  - set:
+      if: "(ctx?.tags.contains('forwarded'))"
```

```suggestion
      if: ctx.tags != null && ctx.tags.contains('forwarded')
```
Similar below (`ctx` is always present).

---

This needs to happen on the two below as well.

---

Done